.github/workflows/test.yml

```diff
@@ -7,26 +7,60 @@ on:
 env:
   apt_options: -o Acquire::Retries=3
   PETSC_DIR: /usr/lib/petscdir/petsc3.15/x86_64-linux-gnu-real
```

Comment:
Can I remove this variable?

Reply:
It is still referenced by the installation of `petsc4py`, IIRC. So I think it should stay in.

Reply:
ok!